README.md

```diff
@@ -60,12 +60,19 @@ Alternatively, you can manually edit Application.xml and then reload the Wowza a
   </Property>
 ```
 
-### To deploy plugin code to VM:
-
-- (ensure Wowza application on VM has the properties above)
-- (update version file)
-- (tag version on github)
-- (trigger jenkins build that creates versioned deployment jar artifact)
+### To deploy a new version of the plugin code to the VM:
+
+- Ensure the Wowza application on the VM has the properties mentioned above.
+- Release a new version of the plugin:
+  - Update `conf/version`.  This should be just the version number itself, e.g. `1.7.1` or `2.0.0-beta`.
+  - Create a version tag and release notes:
```

Comment:
- create the release notes  (in github, go to "code", click on "releases" tab, click on "Draft a new release", create notes (you might look at a diff between current master and the last release tag)
  - "Publish release" will publish the release notes and tag the code